common/src/main/java/feast/common/logging/config/LoggingProperties.java

```diff
@@ -14,19 +14,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package feast.core.exception;
+package feast.common.logging.config;
 
-/** Exception thrown when retrieval of a spec from the registry fails. */
-public class RetrievalException extends RuntimeException {
-  public RetrievalException() {
-    super();
-  }
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
```

Comment:
there was kinda agreement to move from lombok to autovalue, and delete former completely

Reply:
Auto value only provides immutable classes. Spring requires mutable POJOs with setters to inject values into these properties. Without lombok we will have setter hell ala Serving's [FeastProperties](https://github.com/feast-dev/feast/blob/d07875d4efa13c573bffebadf5206d72b7042cf7/serving/src/main/java/feast/serving/config/FeastProperties.java#L199)